common/src/fmt/ui.rs

```diff
@@ -145,6 +146,23 @@ type                    {}",
     }
 }
 
+impl UIfmt for TransactionReceiptWithRevertReason {
+    fn pretty(&self) -> String {
+        format!(
```

Comment:
the reason should only be included if `Some`

so `if let Some(ref reason) self.revert_reason{ do this} else {self.receipt.pretty()}`